wnfs/src/private/directory.rs

```diff
@@ -728,6 +800,21 @@ impl PrivateDirectory {
         forest: &PrivateForest,
         store: &impl BlockStore,
     ) -> Result<Option<PrivateNode>> {
+        if path_segment.is_empty() {
+            let private_node = forest
+                .get(
+                    &self.header.get_private_ref(),
+                    PrivateForest::resolve_lowest,
+                    store,
+                )
+                .await?;
+
+            return Ok(match (search_latest, private_node) {
+                (true, Some(node)) => Some(node.search_latest(forest, store).await?),
+                (_, node) => node,
+            });
+        }
+
```

Comment:
Shouldn't we just have & export a function in `PrivateDirectory` that is called `search_latest`?
Using the empty string here as an indicator seems a little like a hack.

Reply:
I thought it was bit hacky but wanted your input. Thinking there might be situations where that might be useful like with `cd`, `ls` commands. But I agree we should go for exposing `search_latest` for now.

Reply:
For `cd` and `ls`, they should just support handling empty paths then IMO.